### ISSUE TITLE
Confusion about dataset split

### ISSUE THREAD

**Issue by Matilda0224** [yhDing/SPEAR#5]
Hello, thanks for sharing your great work and the code!

I have a question regarding the dataset split in the implementation versus the description in the paper.

- In the paper, it is stated that 20% of nodes are masked out for testing (half for clean accuracy, half for ASR), while from the remaining 80%, 10% are used for labeled training and another 10% for validation. This means the labeled training set should be about 10% of the total nodes.

- However, in the code (see get_split function), the training set is assigned as 20% of all nodes:
                       perm = rs.permutation(data.num_nodes) 
                       train_number = int(0.2*len(perm)) 

This looks different from the paper’s description. Do we need to change this split to reproduce the paper's experiments?

Thanks a lot for clarifying!

**Comment by yhDing** [comment]
Hi! Your understanding of the data split is correct. If you notice any discrepancies between the code and the paper, please follow the paper. Let me know if you have any further questions.
